webview/src/plots/components/App.test.tsx

```diff
@@ -801,13 +801,75 @@ describe('App', () => {
 
     dragEnter(
       anotherSingleViewPlot,
-      'plots-section_template-single_0',
-      DragEnterDirection.RIGHT
+      'template-single_0',
+      DragEnterDirection.LEFT
     )
 
     expect(screen.getByTestId('plot_drop-target')).toBeInTheDocument()
   })
 
+  it('should show a drop target at the end of the section when moving a plot inside of one section but not over any other plot', () => {
```

Comment:
[Q] Should mention template plots? (like the last two tests mention checkpoints)